app/ts/background/settings.ts

```diff
@@ -20,6 +20,61 @@ export const defaultAddresses = [
 	}
 ]
 
+export const defaultRPCs: RPCEntries = [
+	{
+		name: 'Ethereum Mainnet',
+		chainId: 1n,
+		https_rpc: 'https://rpc.dark.florist/flipcardtrustone',
+		currencyName: 'Ether',
+		currencyTicker: 'ETH',
+		primary: true,
+		minimized: true,
+		weth: 0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2n,
+	},
+	{
+		name: 'Goerli',
+		chainId: 5n,
+		https_rpc: 'https://rpc-goerli.dark.florist/flipcardtrustone',
+		currencyName: 'Goerli Testnet ETH',
+		currencyTicker: 'GÖETH',
+		primary: true,
+		minimized: true,
+		weth: 0xb4fbf271143f4fbf7b91a5ded31805e42b2208d6n,
+	},
+	{
+		name: 'Sepolia',
+		chainId: 11155111n,
+		https_rpc: 'https://rpc-sepolia.dark.florist/flipcardtrustone',
+		currencyName: 'Sepolia Testnet ETH',
+		currencyTicker: 'SEETH',
+		primary: true,
+		minimized: true,
+		weth: 0x105083929bf9bb22c26cb1777ec92661170d4285n,
+	},
+	/*
+	{
+		name: 'Eth (geth-multi)',
+		chainId: 1n,
+		https_rpc: 'https://rpc.dark.florist/winedancemuffinborrow',
+		currencyName: 'Ether',
+		currencyTicker: 'ETH',
+		primary: false,
+		minimized: true,
+		weth: 0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2n,
+	},
+	{
+		name: 'Eth (neth-multi)',
+		chainId: 1n,
+		https_rpc: 'https://rpc.dark.florist/birdchalkrenewtip',
+		currencyName: 'Ether',
+		currencyTicker: 'ETH',
+		primary: false,
+		minimized: true,
+		weth: 0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2n,
+	},
+	*/
```

Comment:
Feels like we should leave these in and just discourage their use somehow.  Would rather not have to wait for a plugin update just to troubleshoot them.

Reply:
well, they are completely broken currently as they don't support our multicall method. So I don't think they should be added